bin/www

```diff
@@ -42,12 +43,19 @@ if (key_file_location || cert_file_location) {
        key: fs.readFileSync(key_file_location),
        cert: fs.readFileSync(cert_file_location)
     };
-    server = https.createServer(options, app);
+    server = https.createServer(options, app).listen(https_port, ip);
+
+    http.createServer(function (req, res) {
+        var url_location = "https://" + req.headers['host'].split(":")[0] + ":" + https_port + req.url;
+        res.writeHead(301, { "Location": url_location});
+        res.end();
```

Comment:
Replace above 2 lines with `res.redirect(301, url_location)` instead. It better conveys what's going on in this code.